src/astgen

```diff
@@ -137,11 +137,24 @@ sub read_refs {
 
 #----------------------------------------------------------------------
 
```

Comment:
I suspect these astgen changes aren't intended to be part of this - doesn't seem related.

Reply:
Not really related to the DPI signatures. Just a general developer quality of life thing.. I edited the generated files too often after my editor opend them up with "Go to definition", silly me.

Reply:
I have removed tha changes to astgen from this PR